templates/web/base/questionnaire/index.html

```diff
@@ -21,75 +40,60 @@
 
 <input type="hidden" name="token" value="[% token | html %]">
 
-<p>
-[% loc('The details of your problem are available on the right hand side of this page.') %]
-[% loc('Please take a look at the updates that have been left.') IF updates %]
```

Comment:
A test in [`questionnaire.t`](https://github.com/mysociety/fixmystreet/blob/22c74ad1b5eae2a4c356fb7d35adae762c60488d/t/app/controller/questionnaire.t#L316-L320) relies on this text being present to verify that updates are included on the questionnaire page. Is there some replacement text we can use on L318 to fix the test? ("Last update", or "Show all updates", maybe?)